pylint/interfaces.py

```diff
@@ -46,7 +59,7 @@ def is_implemented_by(cls, instance):
         return implements(instance, cls)
 
 
-def implements(obj: "Interface", interface: Tuple[type, type]) -> bool:
+def implements(obj: "Interface", interface: Type) -> bool:
```

Comment:
I can see one case of `implements` in the code in which this is indeed a `Tuple`. Would this work:
```suggestion
def implements(obj: "Interface", interface: Union[Type["Interface"], Tuple["Interface", ...]]) -> bool:
```

Reply:
Thanks it's a lot better. I guess the idea is to work like is_instance. Another thing is that obj should be Any, right ? We're trying to tell if it's an interface.